robotbase/.vscode/launch.json

```diff
@@ -4,18 +4,24 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
```

Comment:
What caused this file to be modified? It should work the way it was.

Reply:
I'm not sure why it was modified. I didn't mean to commit it.

Reply:
It's still here.

Reply:
The most recent commit to this branch shows the same code in this file as the main code.